Dynavity/Dynavity/model/canvas/CanvasWithAnnotation.swift

```diff
@@ -22,6 +22,6 @@ extension CanvasWithAnnotation: Equatable, Hashable {
     }
 
     public func hash(into hasher: inout Hasher) {
-        hasher.combine(ObjectIdentifier(self).hashValue)
+        hasher.combine(self.name)
```

Comment:
@hansebastian This change is OK right? I made this change because

```
Two instances that are equal must feed the same values to Hasher in hash(into:), in the same order.
```

Reply:
Yep this is okay!